test/itkScancoImageIOTest3.cxx

```diff
@@ -90,7 +90,8 @@ itkScancoImageIOTest3(int argc, char * argv[])
   std::cout << "RescaleUnits: \t\t" << scancoIO->GetRescaleUnits() << std::endl;
   ITK_TEST_EXPECT_EQUAL(scancoIO->GetRescaleUnits(), std::string("mg HA/ccm"));
   std::cout << "CalibrationData: \t\t" << scancoIO->GetCalibrationData() << std::endl;
-  ITK_TEST_EXPECT_EQUAL(scancoIO->GetCalibrationData(), std::string("68 kVp, BH: 200 mg HA/ccm, Scaling 8192, 0.2 CU"));
+  ITK_TEST_EXPECT_EQUAL(scancoIO->GetCalibrationData(), 
```

Comment:
Taking a look at the linter output, it looks like this line was already under the length limit (120?) and should not be modified.

Reply:
Yup, you're right! I'll fix that and upload the changes.